pom.xml

```diff
@@ -38,12 +38,22 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.squareup.okhttp3</groupId>
```

Comment:
in my opinion we should use the webclient class from the  Spring flux library given that we are using Spring boot